kustomizations/apps/data-hub-configs/kubernetes-pipeline.config.yaml

```diff
@@ -38,3 +38,34 @@ kubernetesPipelines:
       - name: data-hub-config-volume
         configMap:
           name: data-hub-configs
+
+  - dataPipelineId: 'Extract_Keywords_From_Corpus_Kubernetes'
+    airflow:
+      dagParameters:
+        schedule: null
+    image: 'docker.io/elifesciences/peerscout-dags:0.0.1'
```

Comment:
I guess we can add some configure to keep the tags updated.
We may also want to keep staging from prod separate. But not sure if we want to maintain two copies of the whole config file.